### DISCUSSION TITLE
Adding a dashboard on startup

### DISCUSSION THREAD

**Discussion by darrigu** [helix-editor/helix#8243] [General]
# Idea

Add a dashboard like [alpha-nvim](https://github.com/goolord/alpha-nvim) on startup:

![](https://user-images.githubusercontent.com/24906808/132604236-4f20adc4-706c-49b4-b473-ebfd6a7f0784.png)

**Comment by archseer** [comment]
I don't find this useful, how is it any better than just opening to an empty scratch buffer?

You can always press space to get a help menu and continue loading files, etc. from there

**Comment by darrigu** [comment]
And the recent projects ? It **is** better, and it **is** useful

**Comment by the-mikedavis** [comment]
A history picker is covered here: https://github.com/helix-editor/helix/issues/2179

**Comment by darrigu** [comment]
Yes but it is not implemented yet

**Comment by Jedsek** [comment]
@archseer
I think it is useful.
For me, if i want to create a new file, I will type `hx file_name` but not `hx`
And `space mode` has more useless options, like `y/Y/p/P/k/r/?/d/w/`
It is better to show a dashboard when just typed `hx` because It could reduce options, provide more intuitive design and greater usability for users.

**Comment by JakeEP** [comment]
I would support this as an option which is on by default. It's nice for "brand recognition" if you like (I know people find this sort of justification distasteful, but that's the way it is). And flagging up the most common options like this would put new users at ease, and new users at ease means they are more likely to stick with it and become long-term users, and more long-term users means more support for Helix, and more long-term users benefits all existing (even power) users.

**Comment by goolord** [comment]
i personally use the `startify` theme because i find more utility in having the `oldfiles` available upon opening nvim

![image](https://user-images.githubusercontent.com/24906808/223852491-e57fa4fc-c597-4696-a85a-a4406adc1603.png)

i have used helix in the past but i'm not sure if i totally grasp its philosophy; imo if i could design a greeter with full support from the text editor i might have some recent files, sessions, etc here

<details>
<summary> fwiw doom emacs and spacemacs also ship with a greeter  </summary>

![image](https://user-images.githubusercontent.com/24906808/223853036-4d971b6d-9c55-4bee-a689-54f5b688e20c.png)

![image](https://user-images.githubusercontent.com/24906808/223853087-4f2c7fda-ad40-4393-a3c7-37149ae0425f.png)

</details>

**Reply by insiwd** [reply]
has there been any update on this feature? i would love to have it
